CourseChamp-Frontend/src/router/index.js

```diff
@@ -42,10 +43,15 @@ export default new Router({
       name: 'CreateCourse',
       component: CreateCourse
     },
+    {
+      path: '/viewreview',
```

Comment:
Don't you need to specify for which course? So

/viewreview/{courseCode}

?

Reply:
No need, this is just to be able to see it, in the future this page will not exist and will be replaced by course/{courseID}